### PR TITLE
Added mobilenet_v1 folder

### PR DESCRIPTION
**Pull request by hitesh-anand**
the file named "mobilenet-v1" was added accidentally.
The main files are inside the folder named "mobilenet_v1"

### CODE REVIEW

File: classification/mobilenet_v1/README.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
+MOBILE NET v1 (TENSORFLOW)
+					(MODEL ZOO)
+
+
```
Comment (SomTambe) [inline classification/mobilenet_v1/README.txt]: Rewrite the README in Markdown. Let it be consistent with the rest of the repo.

File: classification/mobilenet-v1 [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (SomTambe) [inline classification/mobilenet-v1]: Why can't you delete this file?

**Review by SomTambe — CHANGES_REQUESTED** [review]
It was informed well before that jupyter notebooks won't be accepted. Please help us in maintaining consistency with the rest of the repo. Others have done it too.

**Comment by hitesh-anand** [comment]
Deleted the earlier added Jupyter notebook files and README, added a new README written using markdown and a .py file containing the mobilenet model implemented in Google Collab.

**Comment by namangup** [comment]
Look at the pytorch version of MobileNet to get an idea of what is expected. Lot of changes required.

**Pull request closed without merging**